modules/admin_manual/pages/configuration/server/occ_commands/app_commands/richdocuments.adoc

```diff
@@ -0,0 +1,84 @@
+= Collabora Online / Secure View
+
+Marketplace URL: {oc-marketplace-url}/apps/richdocuments[Collabora Online]
+
+Collabora Online allows you to work with all kinds of collabora office documents inside ownCloud for improved productivity. For details please see the
+xref:enterprise/collaboration/collabora_secure_view.adoc[Collabora Online / Secure View] documentation.
+
+`config:app` commands to configure the Collabora Online app.
+
+[source,console]
+----
+ config:app:get|set
+ richdocuments wopi_url            Number of days passed after upload (or restore)
+                                   that files will be archived
+ richdocuments secure_view_option  Number of days passed after archiving that files will
+                                   be permanently deleted
+ richdocuments watermark_text      Define groups of users that are exempt from the
+                                   Lifecycle policies
+ richdocuments secure_view_open_action_default  Restoration Policies for Users
```

Comment:
The column of descriptions talks about archiving, lifecycle, restoration. It seems to have nothing to do with rich documents. Am I missing something?